style.css

```diff
@@ -0,0 +1,118 @@
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+}
+
+.top-header {
+  position: absolute;
+  width: 375px;
```

Comment:
- [OPTIONAL ] You can remove this width so that your webpage can spread all of the webpages, and this will also help your content to fit well in the mobile screen.  Using positioning can work, better flexbox is best here to use.

Reply:
thank you but the reason why i specified the width was to make sure the design is only for mobile devices